experimenter/tests/integration/nimbus/test_cirrus_integration.py

```diff
@@ -158,3 +158,94 @@ def test_create_new_experiment_approve_remote_settings_cirrus(
     # returns the default value
     result_text_element = demo_app.wait_for_result_text(["wicked"])
     assert result_text_element.is_displayed()
+
+
+@pytest.mark.cirrus_enrollment
+def test_check_cirrus_targeting(
```

Comment:
Does all this need to be run for targeting to be checked? Seems like a lot of steps

Reply:
oh I want to make sure in targeting we can select targeting, send for approval, demo app stands up. send the request, and get enrolled and unenroll, like completely end-to-end